lxd/ip/link.go

```diff
@@ -77,6 +77,16 @@ func (l *Link) SetMTU(mtu string) error {
 	return nil
 }
 
+// SetTXQlen sets the txqueuelen of the link device.
+func (l *Link) SetTXQlen(txql string) error {
```

Comment:
This function should take an integer argument.

Reply:
Can we call this `SetTXQueueLength` for clarity please.

Reply:
I renamed function to SetTXQueueLength but in my view it should take string argument as other functions in this file (SetMTU, SetAddress, etc)

Reply:
OK lets use a string for now. Although I don't really like that about the existing functions.